kernel/api/src/main/java/org/sakaiproject/hikaricp/jdbc/pool/SakaiBasicDataSource.java

```diff
@@ -175,7 +175,7 @@ protected void init() throws SQLException
                 } catch (ClassNotFoundException | InstantiationException | IllegalAccessException e) {
                     // TODO Auto-generated catch block
                     String message = "driverClass not specified, might not be able to load the driver'";
-                    M_log.warn(message, e);
+                    M_log.error(message, e);
```

Comment:
If it works without an driver class, why require people to specify extra properties?